apps/vmq_server/src/vmq_info_cli.erl

```diff
@@ -405,6 +409,53 @@ v(_, V) ->
             "\"" ++ V ++ "\""
     end.
 
+get_nested_value([], Map) ->
+    Map;
+get_nested_value([Key | Rest], Map) when is_map(Map) ->
+    case maps:get(Key, Map, undefined) of
+        undefined -> "-";
+        Value -> get_nested_value(Rest, Value)
+    end.
+logger_info(Logger) ->
+    case logger:get_handler_config(Logger) of
+        {ok, Config} ->
+            [
+                {logger, get_nested_value([id], Config)},
+                {level, get_nested_value([level], Config)},
+                {type, get_nested_value([config, type], Config)},
+                {file, get_nested_value([config, file], Config)}
+            ];
+        _ ->
+            []
+    end.
+
+vmq_loq_show_cmd() ->
+    Cmd = ["vmq-admin", "log", "show"],
```

Comment:
I call this differently, `show-config` instead of `show` since show alone implies you are showing some logs. What do you think?

Reply:
What about "show config"? Maybe at some point we might want to show something else as well...

Reply:
Sounds even better